packages/mdc-slider/foundation.js

```diff
@@ -198,10 +190,13 @@ class MDCSliderFoundation extends MDCFoundation {
   }
 
   /**
-   * Called when the user starts interacting with the thumb
+   * Called when the inTransit transition ends
+   * @param {!Event} evt
    */
-  handleThumbPointer() {
-    this.handlingThumbTargetEvt_ = true;
+  handleTransitionEnd(evt) {
+    if (this.inTransit_ && this.adapter_.eventTargetHasClass(evt.target, 'mdc-slider__track-fill')) {
```

Comment:
Add `mdc-slider__track-fill` to constants

Reply:
Done!